JenkinsfileStressTests.groovy

```diff
@@ -55,21 +60,42 @@ pipeline {
+                                    sh 'docker run -d --name=swift -p 8080:8080 -v /srv/bench-running-docker/swift:/srv/1/node/sdb1 jeantil/openstack-keystone-swift:pike'
+                                    sh 'docker run -d --name=rabbitmq -p 15672:15672 -p 5672:5672 rabbitmq:3.8.1-management'
+
+                                    sh 'docker run -d --hostname HOSTNAME -p 25:25 -p 1080:80 -p 8000:8000 -p 110:110 -p 143:143 -p 465:465 -p 587:587 -p 993:993 --link cassandra:cassandra --link rabbitmq:rabbitmq --link elasticsearch:elasticsearch --link tika:tika --link swift:swift --name james_run -t james_run'
+                                    break
+                                case ["s3", "swift"]:
+                                    sh 'docker run -d --name=cassandra -p 9042:9042 cassandra:3.11.3'
+                                    sh 'docker run -d --name=elasticsearch -p 9200:9200 --env "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:6.3.2'
+                                    sh 'docker run -d --name=tika apache/tika:1.24'
+                                    sh 'docker run -d --name=rabbitmq -p 15672:15672 -p 5672:5672 rabbitmq:3.8.1-management'
 
-                            sh 'docker run -d --hostname HOSTNAME -p 25:25 -p 1080:80 -p 8000:8000 -p 110:110 -p 143:143 -p 465:465 -p 587:587 -p 993:993 --link cassandra:cassandra --link rabbitmq:rabbitmq --link elasticsearch:elasticsearch --link tika:tika --link swift:swift --name james_run -t james_run'
+                                    sh 'printenv | grep OS_ > env.file'
+                                    sh 'docker run -d --env-file env.file --hostname HOSTNAME -p 25:25 -p 1080:80 -p 8000:8000 -p 110:110 -p 143:143 -p 465:465 -p 587:587 -p 993:993 --link cassandra:cassandra --link rabbitmq:rabbitmq --link elasticsearch:elasticsearch --link tika:tika --name james_run -t james_run'
+                                    break
+                            }
+                            def jamesCliWithOptions = 'java -jar /root/james-cli.jar -h 127.0.0.1 -p 9999'
                             timeout(time: 20, unit: 'MINUTES') {
                                 retry(200) {
                                     sleep 5
-                                    def jamesCliWithOptions = 'java -jar /root/james-cli.jar -h 127.0.0.1 -p 9999'
                                     sh "docker exec james_run ${jamesCliWithOptions} listusers"
                                 }
                             }
+                            if (params.PROFILE in ["s3", "swift"]) {
+                                sh "docker exec james_run ${jamesCliWithOptions} removedomain localhost"
+                                sh "docker exec james_run ${jamesCliWithOptions} removedomain james.linagora.com"
+                                sh "docker exec james_run ${jamesCliWithOptions} adddomain open-paas.org"
+                                for (int n = 0; n <= 100; n++) {
```

Comment:
do you think we could get these information (`open-paas.org` and `100`) from the gatling scenario? Or maybe it's worth to setup a common env variable?